lib/vzaar_api/category.rb

```diff
@@ -9,17 +9,30 @@ class Category < Abstract
                     :node_video_count, :tree_video_count,
                     :created_at, :updated_at].freeze
 
+    ATTR_ACCESSORS = [:name, :parent_id, :move_to_root].freeze
+
     prepend Lib::HasAttributes
     include Lib::HasResourceUrl
     include Lib::ActiveObject::Find
+    include Lib::ActiveObject::Save
+    include Lib::ActiveObject::Delete
     include Lib::WillPaginate
 
+    def initialize(attrs = {})
+      update_from_attributes attrs
+    end
+
+    def self.create(attrs = {})
+      Strategy::Category::Create.new(attrs, self).execute
```

Comment:
IMO we don't need this `Strategy` class. The only thing it's doing is checking for the presence of `:name` when this is already enforced by the API itself. 

We made a conscious decision to do data validation on the API backend only, so I think this can be removed.